schema/index.md

```diff
@@ -4,24 +4,25 @@ title: "Ehcache XSDs"
 permalink: /schema/
 ---
 
-    
-    
 * Core XSDs
   * [ehcache-core-3.0.xsd](/schema/ehcache-core-3.0.xsd)
   * [ehcache-core-3.1.xsd](/schema/ehcache-core-3.1.xsd)
   * [ehcache-core-3.2.xsd](/schema/ehcache-core-3.2.xsd)
-    
-    
+  * [ehcache-core-3.3.xsd](/schema/ehcache-core-3.3.xsd)
```

Comment:
While you are at it, they should also be added under the `v3` folder in the different `index.md`.